evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -674,12 +674,16 @@ contract FluxAggregator is AggregatorInterface, Owned {
   function roundState(address _oracle)
     external
     view
-    returns (uint32 _reportableRoundId, bool _eligibleToSubmit, int256 _latestRoundAnswer)
+    returns (uint32 _reportableRoundId, bool _eligibleToSubmit, int256 _latestRoundAnswer, uint64 _timesOutAt)
   {
-    uint32 roundId = rounds[reportingRoundId].details.answers.length == rounds[reportingRoundId].details.maxAnswers || timedOut(reportingRoundId)
-      ? reportingRoundId.add(1)
-      : reportingRoundId;
-    return (roundId, eligibleToSubmit(_oracle), rounds[latestRoundId].answer);
+    bool finishedOrTimedOut = rounds[reportingRoundId].details.answers.length == rounds[reportingRoundId].details.maxAnswers || timedOut(reportingRoundId);
```

Comment:
shouldn't this be ">=" as opposed to "=="?

Reply:
Technically it should be impossible for `answers.length` to exceed `maxAnswers`, but you're right, better to be safe.